src/http-driver.js

```diff
@@ -103,14 +108,12 @@ function makeHTTPDriver({eager = false} = {eager: false}) {
       .map(reqOptions => {
         let response$ = createResponse$(reqOptions)
         if (eager || reqOptions.eager) {
-          response$ = response$.replay(null, 1)
-          response$.connect()
+          response$ = response$.publishReplay(null, 1).connect()
+          return response$
         }
         response$.request = reqOptions
         return response$
-      })
-      .replay(null, 1)
-    response$$.connect()
+      }).shareReplay(null, 1)
```

Comment:
response$$ wth `publishReplay(1)` + `connect()` 100% does **not** work. Yes I realized that it shouldn't have a an observable as it's value, using and empty `{}` seems to work though.